src/pybind/mgr/cephadm/migrations.py

```diff
@@ -81,34 +86,57 @@ def verify_no_migration(self) -> None:
                 "cephadm migration still ongoing. Please wait, until the migration is complete.")
 
     def migrate(self, startup: bool = False) -> None:
+
+        logger.info('running migrations')
+
         if self.mgr.migration_current == 0:
+            logger.info('trying migrate_0_1')
             if self.migrate_0_1():
+                logger.info('migrate_0_1 Done')
                 self.set(1)
 
         if self.mgr.migration_current == 1:
+            logger.info('trying migrate_1_2')
             if self.migrate_1_2():
+                logger.info('migrate_1_2 Done')
                 self.set(2)
 
         if self.mgr.migration_current == 2 and not startup:
+            logger.info('trying migrate_2_3')
             if self.migrate_2_3():
+                logger.info('migrate_2_3 Done')
                 self.set(3)
 
         if self.mgr.migration_current == 3:
+            logger.info('trying migrate_3_4')
             if self.migrate_3_4():
+                logger.info('migrate_3_4 Done')
                 self.set(4)
 
         if self.mgr.migration_current == 4:
+            logger.info('trying migrate_4_5')
             if self.migrate_4_5():
+                logger.info('migrate_4_5 Done')
                 self.set(5)
 
         if self.mgr.migration_current == 5:
+            logger.info('trying migrate_5_6')
             if self.migrate_5_6():
+                logger.info('migrate_5_6 Done')
                 self.set(6)
 
         if self.mgr.migration_current == 6:
+            logger.info('trying migrate_6_7')
             if self.migrate_6_7():
+                logger.info('migrate_6_7 Done')
                 self.set(7)
 
+        if self.mgr.migration_current == 7:
+            logger.info('trying migrate_7_8')
+            if self.migrate_7_8():
+                logger.info('migrate_7_8 Done')
+                self.set(8)
```

Comment:
given these are all identical other than the number, I'm wondering if there is some way to sort of automate this bit where it automatically loops from the current `migration_current` up to whatever `LAST_MIGRATION` is and handles logging the right numbers and calling the migration functions with the right numbers so we don't need to keep worrying about this. Probably not a task for this PR, but something to think about